vendor/bat-native-ads/src/bat/ads/internal/frequency_capping/exclusion_rules/anti_targeting_frequency_cap.cc

```diff
@@ -33,17 +34,17 @@ bool HasVisitedSiteOnAntiTargetingList(
 }  // namespace
 
 AntiTargetingFrequencyCap::AntiTargetingFrequencyCap(
-    resource::AntiTargeting* anti_targeting,
-    const BrowsingHistoryList& history)
-    : anti_targeting_(anti_targeting), history_(history) {}
+    resource::AntiTargeting* anti_targeting_resource,
+    const BrowsingHistoryList& browsing_history)
+    : anti_targeting_resource_(anti_targeting_resource),
+      browsing_history_(browsing_history) {}
 
 AntiTargetingFrequencyCap::~AntiTargetingFrequencyCap() = default;
 
 bool AntiTargetingFrequencyCap::ShouldExclude(const CreativeAdInfo& ad) {
   if (!DoesRespectCap(ad)) {
     last_message_ = base::StringPrintf(
-        "creativeSetId %s excluded as user has "
-        "previously visited an anti-targeting site",
+        "creativeSetId %s excluded as previously visited an anti-targeted site",
```

Comment:
not specific to this PR, but do these messages get logged/written somewhere?

Reply:
These messages get logged to the diagnostic log for Brave Rewards.

Reply:
@tmancey got it, so they're only logged if the user runs brave with the logging flag?

Reply:
@diracdeltas That is correct, thanks